wordpress-mobile/gutenberg-mobile
# Merge 1.43.0 release back to the development branch after resolving conflicts

This PR was needed because there are conflicts on the `trunk` → `develop` branch here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2909

To avoid pushing straight to `trunk`, a new branch is made here called `merge_release_1.43.0_to_develop`, `develop` is merged into it, conflicts are resolved, and this PR merges this to `develop`. This has the result of updating `develop` with the release that's in `trunk`.

### To test
Check that the Gutenberg submodule ref points to https://github.com/WordPress/gutenberg/commit/a454634b3ebacc9175a22c266142ab2ef985454a, as described in https://github.com/wordpress-mobile/gutenberg-mobile/pull/2909#pullrequestreview-557465394.

PR submission checklist:
